least_common_paths.txt

```diff
@@ -0,0 +1,992 @@
+{'altair': {'common_path': '.',
+            'files': ['./docs_rtd/example_code/custom_renderer.py',
+                      './great_expectations/expectations/core/expect_column_distinct_values_to_be_in_set.py',
+                      './great_expectations/expectations/core/expect_column_kl_divergence_to_be_less_than.py',
+                      './great_expectations/rule_based_profiler/altair/encodings.py',
+                      './great_expectations/rule_based_profiler/data_assistant_result/data_assistant_result.py',
+                      './great_expectations/rule_based_profiler/data_assistant_result/plot_components.py',
+                      './great_expectations/rule_based_profiler/data_assistant_result/plot_result.py',
+                      './tests/rule_based_profiler/data_assistant/test_data_assistant_result.py',
+                      './tests/rule_based_profiler/data_assistant/test_onboarding_data_assistant.py',
+                      './tests/rule_based_profiler/data_assistant/test_volume_data_assistant.py'],
+            'requirements_file': 'requirements.txt'},
```

Comment:
For each requirement (dictionary key), we get the following:
* The files that requirement is imported in
* The actual requirements.txt file that requirement is defined in
* The common path for the usage of that requirement